test/database_service/test_experiment_data_integration.py

```diff
@@ -0,0 +1,517 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020, 2021
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Experiment integration tests."""
+
+import os
+from unittest import mock, SkipTest, skipIf
+import contextlib
+import numpy as np
+
+from qiskit import transpile
+from qiskit.providers import JobStatus
+from qiskit.test.reference_circuits import ReferenceCircuits
+from qiskit.tools.visualization import HAS_MATPLOTLIB
+
+from qiskit.providers.ibmq.experiment import (
+    IBMExperimentService,
+    ResultQuality,
+    IBMExperimentEntryNotFound,
+)
+
+from qiskit.test import QiskitTestCase
+from ..decorators import requires_provider, requires_device
+
+try:
+    from qiskit_experiments.database_service import DbExperimentDataV1 as DbExperimentData
+    from qiskit_experiments.database_service import DbAnalysisResultV1 as AnalysisResult
+    from qiskit_experiments.database_service.exceptions import DbExperimentEntryNotFound
+
+    HAS_QISKIT_EXPERIMENTS = True
+except ImportError:
+    HAS_QISKIT_EXPERIMENTS = False
```

Comment:
This block and `HAS_QISKIT_EXPERIMENTS` can be removed since you're in the experiments package now